typescript/.changeset/six-moons-begin.md

```diff
@@ -0,0 +1,5 @@
+---
+"@coinbase/cdp-sdk": minor
+---
+
+Added getSmartAccount, createSmartAccount, and getOrCreateSmartAccount methods to the EVM client
```

Comment:
`getSmartAccount` and `createSmartAccount` already existed. How about sthg like:
```
Added getOrCreateSmartAccount by name to the EVM client.

Also updated getSmartAccount to allow getting a smart account by name and createSmartAccount to allow creating a smart account with a name.
```